layers/boundaries.py

```diff
@@ -0,0 +1,200 @@
+    ensureEmptyTempDirectoryExists(tmp)
+
+    # Get the geopackage
+    run(
+        [
+            "wget",
+            # From https://osdatahub.os.uk/downloads/open/BoundaryLine
+            "https://api.os.uk/downloads/v1/products/BoundaryLine/downloads?area=GB&format=GeoPackage&redirect",
+            "-O",
+            f"{tmp}/boundary_lines.zip",
+        ]
+    )
+    run(["unzip", f"{tmp}/boundary_lines.zip", "-d", tmp])
+
+    # Convert to GeoJSON, projecting to WGS84. Only grab one layer.
+    run(
+        [
+            "ogr2ogr",
+            "-f",
+            "GeoJSON",
+            f"{tmp}/parliamentary_constituencies.geojson",
+            "-t_srs",
+            "EPSG:4326",
+            f"{tmp}/Data/bdline_gb.gpkg",
+            "-sql",
+            # Just get a few fields from one layer, and filter for England
+            "SELECT Name, Census_Code, geometry FROM westminster_const WHERE Census_Code LIKE 'E%'",
+        ]
+    )
+
+    convertGeoJsonToPmtiles(
+        f"{tmp}/parliamentary_constituencies.geojson",
+        "output/parliamentary_constituencies.pmtiles",
+    )
+
+
+# You have to manually download the GeoJSON file from https://geoportal.statistics.gov.uk/datasets/ons::wards-may-2023-boundaries-uk-bgc/explore and pass in the path here (until we can automate this)
+def makeWards(path):
+    tmp = "tmp_wards"
+    ensureEmptyTempDirectoryExists(tmp)
+
+    # Clean up the file
+    print(f"Cleaning up {path}")
+    gj = {}
+    with open(path) as f:
+        gj = json.load(f)
+        # Remove unnecessary attributes
+        del gj["name"]
+        del gj["crs"]
+
+        # Only keep England
+        gj["features"] = list(
+            filter(lambda f: f["properties"]["WD23CD"][0] == "E", gj["features"])
+        )
+
+        for feature in gj["features"]:
+            # Remove most properties, and rename a few
+            props = {}
+            props["WD23CD"] = feature["properties"]["WD23CD"]
+            props["name"] = feature["properties"]["WD23NM"]
+            feature["properties"] = props
+
+            feature["geometry"]["coordinates"] = trimPrecision(
+                feature["geometry"]["coordinates"]
+            )
+    with open(f"{tmp}/wards.geojson", "w") as f:
+        f.write(json.dumps(gj))
+
+    convertGeoJsonToPmtiles(f"{tmp}/wards.geojson", "output/wards.pmtiles")
+
+
+def makeCombinedAuthorities():
+    tmp = "tmp_combined_authorities"
+    ensureEmptyTempDirectoryExists(tmp)
+
+    # Reproject to WGS84
+    run(
+        [
+            "ogr2ogr",
+            "-f",
+            "GeoJSON",
+            f"{tmp}/boundary.geojson",
+            "-t_srs",
+            "EPSG:4326",
+            # Manually downloaded and stored in git
+            "input/Combined_Authorities_December_2022_EN_BUC_1154653457304546671.geojson",
+        ]
+    )
+
+    # Clean up the file. Note the features already have IDs.
+    print(f"Cleaning up {tmp}/boundary.geojson")
```

Comment:
There are some more places we could use the helper method, like this. I might later add options to filter out features (for some of the census stuff) and clean up top-level properties like crs